client/src/components/main/EditPassword.jsx

```diff
@@ -9,6 +9,11 @@ const EditPassword = (props) => {
   const [password, setPassword] = useState('');
   const [message, setMessage] = useState('');
 
+  //only on first render
+  useEffect(()=>{
+    setWebsiteName(item);
+  },[])
+  
   function updateFunc(){
     if(websiteName && password){
```

Comment:
Add input validation check for website name and password

Reply:
Okk

Reply:
Can you please tell me the validation rules for the website name and password.
I think we don't need to add validation to the password because different websites have different Validation rules 
client just write and save their passwords

Reply:
Ok then leave it for now